src/App.js

```diff
@@ -55,11 +138,29 @@ const App = () => {
       </header>
       <div style={{ display: 'flex', width: '100%' }}>
         <div style={{ width: '25%' }}>
-          Marker List
-          {/* Add list of todos here */}
+          <h3>Marker List</h3>
+          {markers?.length > 0 ? markers.map((marker) => (
+            <div onClick={() => viewMarker(marker.title)} name={marker.title} style={{ display: 'flex', justifyContent: 'space-between' }}>
```

Comment:
When using Array.map(), each child in a list should have a unique 'key' prop. This can be solved by:
```
{markers?.length > 0 ? markers.map((marker,index)=>{
<div key={index} onClick={...}>
// the rest of your code goes here...
```